src/client/objectTypes/objectTypeActions.js

```diff
@@ -61,6 +62,33 @@ export const getObjectType = (typeName, { limit = 15, skip = 0 } = { limit: 15,
   });
 };
 
+export const getObjectTypeMap = ({ limit = 30, skip = 0, map = {} } = { limit: 30, skip: 0, map: {} }) => (
```

Comment:
duplicated code (with previous action). Need to refactor

Reply:
changed